packages/tool-cache/src/tool-cache.ts

```diff
@@ -200,6 +200,10 @@ export async function extractTar(
   }
 
   dest = dest || (await _createExtractFolder(dest))
+  if (IS_WINDOWS) {
+    // For some reason, GNU Tar on Windows expects paths to be slash-separated even though this is not the standard Windows path delimiter
+    dest = dest.replace(/\\/g, '/')
```

Comment:
I proposed an alternate solution for https://github.com/actions/toolkit/pull/165 which will eliminate the need for this.

Let's close on the other PR first before proceeding with this PR.